config/env.go

```diff
@@ -0,0 +1,35 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+package config
+
+import "os"
+
+var Config = newEnvConfig()
+
+type envConfig struct {
+	ServerURL   string
+	AccessToken string
+	Workflow    string
+}
+
+func newEnvConfig() *envConfig {
+	config := &envConfig{
+		ServerURL:   envOrDefault("REANA_SERVER_URL", ""),
```

Comment:
This is the same as `os.GetEnv()` but I think this is more clear and we can use the same logic for other default values in the future